tests/yam/migration/migration_unattended.pm

```diff
@@ -45,6 +45,9 @@ sub run {
         }
     }
 
+    # bump grub timeout for migration image
+    assert_script_run("sed -i 's/set timeout=[0-9]*/set timeout=-1/' /etc/grub.d/99_migration && grub2-mkconfig -o /boot/grub2/grub.cfg") unless is_s390x();
```

Comment:
Please split it in two openQA commands for easier disecction.
Additionally move this logic to s390 condition below to minimize the conditional sentences.

Reply:
updated, thanks.